indexing-service/src/test/java/org/apache/druid/indexing/common/actions/SegmentAllocationQueueTest.java

```diff
@@ -99,6 +121,133 @@ public void tearDown()
     emitter.flush();
   }
 
+  @Test
+  @Ignore
```

Comment:
Let's remove this test if we are not running it as a UT. If it is meant to be a benchmark, please put it in a `Benchmark` class and share the details of a some sample run in the PR description.

Reply:
Understood, will remove it.